site/_resources/upgrading.md

```diff
@@ -753,3 +824,4 @@ $ kubectl get configmap -n heptio-contour -o yaml contour
 [20]: https://www.envoyproxy.io/docs/envoy/v1.14.1/intro/version_history
 [21]: https://www.envoyproxy.io/docs/envoy/v1.14.2/intro/version_history
 [22]: https://www.envoyproxy.io/docs/envoy/v1.14.3/intro/version_history
+[23]: https://www.envoyproxy.io/docs/envoy/v1.15.0/version_history/current
```

Comment:
Is this the long-term link for v1.15?

Reply:
Not sure what you mean by long-term, but using the existing link format that we use -- https://www.envoyproxy.io/docs/envoy/v1.15.0/intro/version_history -- indicates that release notes have moved to https://www.envoyproxy.io/docs/envoy/v1.14.3/intro/version_history, so I'm just using the latter.

Reply:
Oh I see they moved it's location. =)